Recrutify/Recrutify.Services/Validators/UpdateProjectValidator.cs

```diff
@@ -35,6 +35,11 @@ private void ConfigureRules()
         private async Task UmmutableFieldsAreKeepingUnchanged(UpdateProjectDTO dto, ValidationContext<UpdateProjectDTO> context, CancellationToken cancellationToken)
         {
             var project = await _projectRepository.GetAsync(dto.Id);
+            if (project == null)
+            {
+                throw new ValidationException("Primary skill does't exist!");
```

Comment:
why you dont't write  context.AddFailure like in other places of this method??

Reply:
validator has existing mechanisms for adding errors, why you trigger exception here

Reply:
done